atomicapp/constants.py

```diff
@@ -61,3 +61,5 @@
 PROVIDER_CONFIG_KEY = "providerconfig"
 
 PROVIDERS = ['docker', 'kubernetes', 'openshift']
+PROVIDER_API_KEY = "providerapi"
```

Comment:
We already discussed it here https://github.com/projectatomic/atomicapp/pull/392#discussion_r44532586 :-)
I was also thinking about `providerendpoint` and `providerurl`, but first is longer :-) and second is not good.